reconcile/slack_usergroups_api.py

```
@@ -210,7 +224,7 @@ def compile_users_from_schedule(
             start = ensure_utc(datetime.strptime(entry.start, DATE_FORMAT))  # ruff: ignore[call-datetime-strptime-without-zone]
```

Ruff (0.16.0)

[warning] Invalid rule code in suppression: call-datetime-strptime-without-zone

Enable `lint.preview` to use rule names

(RUF102)